bigtable-dataflow-parent/bigtable-dataflow-import/src/main/java/com/google/cloud/bigtable/dataflowimport/HBaseExportJob.java

```diff
@@ -0,0 +1,303 @@
+import com.google.cloud.dataflow.sdk.transforms.Sum.SumLongFn;
+import com.google.cloud.dataflow.sdk.transforms.display.DisplayData;
+import com.google.cloud.dataflow.sdk.util.Reshuffle;
+import com.google.cloud.dataflow.sdk.values.KV;
+import com.google.cloud.dataflow.sdk.values.TypeDescriptor;
+import com.google.common.base.Joiner;
+import com.google.common.base.Objects;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hbase.client.Result;
+import org.apache.hadoop.hbase.client.Scan;
+import org.apache.hadoop.hbase.filter.ParseFilter;
+import org.apache.hadoop.hbase.io.ImmutableBytesWritable;
+import org.apache.hadoop.hbase.mapreduce.ResultSerialization;
+import org.apache.hadoop.io.SequenceFile;
+import org.apache.hadoop.io.SequenceFile.CompressionType;
+import org.apache.hadoop.io.serializer.WritableSerialization;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * <p>
+ *  Dataflow job to export a Bigtable table to a set of SequenceFiles.
+ *  Afterwards, the files can be either imported into another Bigtable or HBase table.
+ *  You can limit the rows and columns exported using the options in {@link HBaseExportOptions}.
+ *  Please note that the pipeline processes the rows in chunks rather than rows, so the element counts in
+ *  dataflow ui will be misleading.
+ *
+ *  Example usage:
+ * </p>
+ *
+ * <pre>
+ * {@code mvn compile exec:java \
+ *    -Dexec.mainClass=com.google.cloud.bigtable.dataflowimport.HBaseExportJob \
+ *    -Dexec.args="--runner=BlockingDataflowPipelineRunner \
+ *    --project=igorbernstein-dev \
+ *    --stagingLocation=gs://igorbernstein-dev/dataflow-staging \
+ *    --zone=us-east1-c \
+ *    --bigtableInstanceId=instance1 \
+ *    --bigtableTableId=table2 \
+ *    --destination=gs://igorbernstein-dev/export55 \
+ *    --maxNumWorkers=200"
+ * }
+ * </pre>
+ */
+public class HBaseExportJob {
+  private static final Logger LOG = LoggerFactory.getLogger(HBaseExportJob.class);
+
+  public static void main(String[] args) throws IOException {
+    PipelineOptionsFactory.register(HBaseExportOptions.class);
+
+    final HBaseExportOptions options = PipelineOptionsFactory.fromArgs(args)
+        .withValidation()
+        .as(HBaseExportOptions.class);
+
+    final Pipeline pipeline = buildPipeline(options);
+    pipeline.run();
+  }
+
+  /**
+   * Builds a pipeline using the passed in options.
+   */
+  public static Pipeline buildPipeline(HBaseExportOptions options) throws IOException {
+    if(options.getDiskSizeGb() == 0) {
+      options.setDiskSizeGb(20);
+    }
+    if(options.getBigtableProjectId() == null) {
+      options.setBigtableProjectId(options.getProject());
+    }
+
+    // Configure Bigtable scan parameters
+    final Scan scan = new Scan()
```

Comment:
To avoid overlap, how about `buildPipeline(DataflowOptions, CloudBigtableScanConfiguration, String destination)`?